content/sponsors.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Sponsors For TARDIS
+seo:
+  title: Sponsors for TARDIS
+  description: Sponsors for TARDIS Organisation
+  extra:
+    - name: 'og:type'
+      value: website
+      keyName: property
+    - name: 'og:title'
+      value: CodeOfConduct
+      keyName: property
+    - name: 'og:description'
+      value: Code Of Conduct for TARDIS Organisation
```

Comment:
What should this be? I fixed a typo, but otherwise I'm not sure. I commented out everything in the front matter but the title and layout, and nothing changed.

Reply:
It's part of sponsors page so `value`s should be for Sponsors not Code of Conduct